en/concepts/scene/light/shadow.md

```diff
@@ -36,7 +36,7 @@ The Planar shadow type is generally used for simpler scenes.
 | :--- | :--- |
 | **Enabled** | Whether to enable shadow effect |
 | **Type** | Shadow type |
-| **Saturation**  | Set the shadow saturation  |
+| **Saturation**  | Set the shadow saturation, it is recommended to set as **1**. If need to reduce the saturation of the directional light shadows, we recommend to do it by increasing the ambient light instead of adjusting this value  |
```

Comment:
```suggestion
| **Saturation**  | Set the shadow saturation, it is recommended to set it as **1**. If it is necessary to reduce the saturation of the directional light shadows, it is recommended to do it by increasing the ambient light instead of adjusting this value  |
```